pandas/core/frame.py

```diff
@@ -4818,66 +4818,82 @@ def aggregate(self, func, axis=0, *args, **kwargs):
-            apply will use reduce to determine whether the result should be a
-            Series or a DataFrame. If reduce is None (the default), apply's
-            return value will be guessed by calling func an empty Series (note:
-            while guessing, exceptions raised by func will be ignored). If
-            reduce is True a Series will always be returned, and if False a
-            DataFrame will always be returned.
-
-            .. deprecated:: 0.23.0
+            Try to apply reduction procedures. If the `DataFrame` is empty,
+            :meth:`apply` will use reduce to determine whether the result
+            should be a `Series` or a `DataFrame`. If reduce is None (the
+            default), :meth:`apply`'s return value will be guessed by calling
+            func on an empty `Series`
+            (note: while guessing, exceptions raised by `func` will be
+            ignored).
+            If reduce is True a `Series` will always be returned, and if
+            `False` a `DataFrame` will always be returned.
+
+            .. deprecated:: 0.23.0.
                This argument will be removed in a future version, replaced
                by result_type='reduce'.
 
-        result_type : {'expand', 'reduce', 'broadcast, None}
-            These only act when axis=1 {columns}:
+        result_type : {'expand', 'reduce', 'broadcast', None}
+            These only act when `axis=1` (columns):
 
             * 'expand' : list-like results will be turned into columns.
-            * 'reduce' : return a Series if possible rather than expanding
-              list-like results. This is the opposite to 'expand'.
+            * 'reduce' : returns a `Series` if possible rather than expanding
+              list-like results. This is the opposite of 'expand'.
             * 'broadcast' : results will be broadcast to the original shape
-              of the frame, the original index & columns will be retained.
+              of the `DataFrame`, the original index and columns will be
+              retained.
 
-            The default behaviour (None) depends on the return value of the
-            applied function: list-like results will be returned as a Series
-            of those. However if the apply function returns a Series these
+            The default behaviour (`None`) depends on the return value of the
+            applied function: list-like results will be returned as a `Series`
+            of those. However if the apply function returns a `Series` these
             are expanded to columns.
 
-            .. versionadded:: 0.23.0
+            .. versionadded:: 0.23.0.
 
         args : tuple
             Positional arguments to pass to function in addition to the
-            array/series
-        Additional keyword arguments will be passed as keywords to the function
+            array/series.
+        kwds :
+            Additional keyword arguments to pass as keywords to the function.
```

Comment:
A personal opinion, but for me it'd be a bit more explicit to say "to pass as keyword arguments to `func`". I think saying "the function" should be clear enough for most people, but specifying which functions avoids any ambiguity. Also por `args`.

Reply:
add this PR review suggestions into commit 6cbadd8ed